bin/data-push.js

```diff
@@ -90,11 +90,11 @@ Promise.resolve().then(async () => {
       },
       schedule: argv.schedule
     }
-    await datahub.push(dataset, options)
-
+    let res = await datahub.push(dataset, options)
```

Comment:
we should handle error here. What if it fails? 
Also, it might return `res.success: false` - if so we should display error

Reply:
Also `const` not let

Reply:
It was handled in `datahub-client`, why we need to double check errors? only success message returns